waspc/cli/Wasp/Cli/Command/Build.hs

```diff
@@ -35,16 +35,16 @@ build = do
 
   doesBuildDirExist <- liftIO $ doesDirectoryExist buildDirFilePath
   when doesBuildDirExist $ do
-    waspSaysC $ asWaspStartMessage "Clearing the content of the .wasp/build directory..."
+    cliSendMessageC $ Msg.Start "Clearing the content of the .wasp/build directory..."
     liftIO $ removeDirectoryRecursive buildDirFilePath
-    waspSaysC $ asWaspSuccessMessage "Successfully cleared the contents of the .wasp/build directory."
+    cliSendMessageC $ Msg.Success "Successfully cleared the contents of the .wasp/build directory."
 
-  waspSaysC $ asWaspStartMessage "Building wasp project..."
+  cliSendMessageC $ Msg.Start "Building wasp project..."
   buildResult <- liftIO $ buildIO waspProjectDir buildDir
   case buildResult of
-    Left compileError -> throwError $ CommandError $ asWaspFailureMessage "Build failed:" ++ compileError
-    Right () -> waspSaysC $ asWaspSuccessMessage "Code has been successfully built! Check it out in .wasp/build directory."
-  waspSaysC alphaWarningMessage
+    Left compileError -> throwError $ CommandError $ "Build failed:" ++ compileError
+    Right () -> cliSendMessageC $ Msg.Success "Code has been successfully built! Check it out in .wasp/build directory."
+  cliSendMessageC $ Msg.Warning alphaWarningMessage
```

Comment:
Does `alphaWarningMessage` already contain some styling? Might that be a problem? Because I see that before it was directly used with `waspSaysC`.

Reply:
It was just a string, so now it's an actual warning. I think that makes sense as it's labeled "warning message".